src/CoreBundle/MaintenanceMode/MaintenanceModeService.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\MaintenanceMode;
+
+use ChameleonSystem\CoreBundle\Exception\MaintenanceModeErrorException;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DBALException;
+use esono\pkgCmsCache\CacheInterface;
+
+class MaintenanceModeService implements MaintenanceModeServiceInterface
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    /**
+     * @var CacheInterface
+     */
+    private $cache;
+
+    public function __construct(Connection $connection, CacheInterface $cache)
+    {
+        $this->connection = $connection;
+        $this->cache = $cache;
+    }
+
+    public function isActivated(): bool
```

Comment:
Now this method and `isActivatedInDb` are public in the implementation but gone in the interface. What's the intention behind this (you mentioned a follow-up PR, but I can't see the big plan - PRs should be meaningful for themselves)? The implementation should not have additional public methods.

Reply:
Only alternative would be to remove these methods now and re-add them with the next to come PR.

Removing them from the interface was the idea to "not break the interface" with the next PR.

Reply:
We're running out of time for the current release, so I suggest to move this PR to 6.2.4, so that we can manage both PRs at once in case they depend on each other (as long as an interface is not published in a release we can change it anytime).